packages/@sanity/form-builder/src/inputs/ImageInput/SelectAsset.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import client from 'part:@sanity/base/client'
 import Button from 'part:@sanity/components/buttons/default'
 import styles from './styles/SelectAsset.css'
-import {get} from 'lodash'
+import ImageAsset from './Asset'
```

Comment:
A bit of inconsistent naming going on here

Reply:
👍

Reply:
Hope this does not conflict with the ts type Asset definition under. Seems to compile without errors. Maybe the es-linter is confused by Typescript inside an js file.